.github/scripts/trymerge.py

```diff
@@ -1983,7 +1992,8 @@ def handle_exception(e: Exception, title: str = "Merge failed") -> None:
         message += '\nIf those updates are intentional, please add "submodule" keyword to PR title/description.'
         gh_post_pr_comment(org, project, args.pr_num, message, dry_run=args.dry_run)
         return
-
+    if "merging" not in pr.get_labels():
+        pr.add_numbered_label("merging")
```

Comment:
Please do not hardcode string constants (because in the long run, somebody makes a typo which would be very hard to debug)
Also, if you are checking that label is not there, why not call `gh_add_labels` directly?
```suggestion
    if MERGE_IN_PROGRESS not in pr.get_labels():
        gh_add_label(org, project, args.pr_num, [MERGE_IN_PROGRESS])
```